### PR TITLE
Update migration guide

### PR DESCRIPTION
**Pull request by m-vdb**
**Proposed changes**:
- Fixes https://github.com/RasaHQ/rasa/issues/6863
- Documents https://github.com/RasaHQ/rasa/pull/6854

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by m-vdb** [comment]
@erohmensing what did you mean by "The migration before and after changes should have been explained with an example" in the original ticket? I'm not sure what it refers to

**Comment by erohmensing** [comment]
@m-vdb these came bullets right from your 2.0 feedback notes 😉

I assume when migrating things, showing an example -- for example the names of the parameters from two stage fallback to rulepolicy are a bit different -- an example might help there

**Comment by m-vdb** [comment]
🤔 then it might relate to older versions of the docs, because the migration guide currently includes example for each of the policies to migrate 🤷

File: docs/docs/migration-guide.mdx [hunk 1]
```diff
@@ -57,34 +57,59 @@ deprecated:
 - [Two-Stage-Fallback Policy](./policies.mdx#two-stage-fallback-policy)
 - [Form Policy](./policies.mdx#form-policy)
 
-#### Migrating from the Mapping Policy
+:::tip Automatic migration
+Rasa Open Source can take care of automatically migrating the Mapping Policy,
```
Comment (wochinge) [inline docs/docs/migration-guide.mdx:61]: should we add links to their deprecated pendants in the policy docs?

File: docs/docs/migration-guide.mdx [hunk 2]
```diff
@@ -57,34 +57,59 @@ deprecated:
...
+  - name: RulePolicy
+```
+
+#### Manually migrating from the Fallback Policy
 
 If you previously used the [Fallback Policy](./policies.mdx#fallback-policy), the following model
 configuration would translate as follows given a previous configuration like this:
 
-```yaml-rasa
+```yaml-rasa title="config.yml"
```
Comment (wochinge) [inline docs/docs/migration-guide.mdx:112]: ❤️

File: docs/docs/migration-guide.mdx [hunk 3]
```diff
@@ -95,12 +120,12 @@ policies:
...
 The new configuration would then look like:
 
-```yaml-rasa
+```yaml-rasa title="config.yml"
 policies:
-# Other policies
-- name: RulePolicy
-  core_fallback_threshold: 0.3
-  core_fallback_action_name: "action_default_fallback"
+  # Other policies
```
Comment (wochinge) [inline docs/docs/migration-guide.mdx:125]: do we have an convention how to indent yaml in the docs? We usually don't add the 2 spaces afaik

File: docs/docs/migration-guide.mdx (hunk 1 above)
Comment (m-vdb) [reply docs/docs/migration-guide.mdx:61]: the links are in the list above. I don't think it adds much TBH, and rather clutters the callout.

File: docs/docs/migration-guide.mdx [hunk 4]
```diff
@@ -57,34 +57,59 @@ deprecated:
...
 rules:
 - rule: Rule to map `ask_is_bot` intent
   steps:
   - intent: ask_is_bot
   - action: action_is_bot
 ```
 
-#### Migrating from the Fallback Policy
+After removing the triggers from your domain file, you can finally replace the
+Mapping Policy with the [Rule Policy](./policies.mdx#rule-policy) in your model configuration:
```
Comment (wochinge) [inline docs/docs/migration-guide.mdx:99]: should we give an example of how the domain looks without `triggers:`?

File: docs/docs/migration-guide.mdx (hunk 3 above)
Comment (m-vdb) [reply docs/docs/migration-guide.mdx:125]: I took a look at the config files in the repository and most of them have the 2-space indent. We do not have linting enforcing this at the moment, but we could, I think.

File: docs/docs/migration-guide.mdx (hunk 4 above)
Comment (m-vdb) [reply docs/docs/migration-guide.mdx:99]: yep why not

File: docs/docs/migration-guide.mdx [hunk 5]
```diff
@@ -57,34 +57,66 @@ deprecated:
...
+```yaml-rasa title="rules.yml"
 rules:
 - rule: Rule to map `ask_is_bot` intent
   steps:
   - intent: ask_is_bot
   - action: action_is_bot
 ```
 
-#### Migrating from the Fallback Policy
+And you can safely remove any trigger from your domain:
```
Comment (wochinge) [inline docs/docs/migration-guide.mdx:98]:
```suggestion
And you can safely remove any `triggers:` from your domain:
```

File: docs/docs/migration-guide.mdx [hunk 6]
```diff
@@ -159,16 +191,16 @@ pipeline:
 In addition you need to add a [Rule](./rules.mdx) to activate the Two-Stage Fallback for
 messages with low NLU confidence.
 
-```yaml-rasa
+```yaml-rasa title="rules.yml"
```
Comment (wochinge) [inline docs/docs/migration-guide.mdx:194]: would you please be so kind and also add the files `title` to  the ones for migrating the `FormPolicy`?

**Review by wochinge — APPROVED** [review]
💯

**Comment by akelad** [comment]
oops i also started reviewing this, let me just see if any of my comments are still valid

**Review by akelad — COMMENTED** [review]
only one comment :D

**Pull request merged**